app/controllers/users_controller.rb

```diff
@@ -0,0 +1,24 @@
+class UsersController < ApplicationController
+
+  def index
+    @users = User.all
+  end
+
+  def create
+    @user = User.new(user_params)
+
+    if @user.save
+      redirect_to @user, notice: "You're in!"
+    else
+      render action: 'sign_up'
+    end
+  end
+
+private
```

Comment:
Indent access modifiers like `private`.<br>Keep a blank line before and after `private`.